### ISSUE TITLE
Angular material cdn links for  js and css showing certificate errors

### ISSUE THREAD

**Issue by murtuzamj** [schme16/gitcdn.xyz#82]
https://cdn.gitcdn.link/cdn/angular/bower-material/v1.1.5/angular-material.css

https://cdn.gitcdn.link/cdn/angular/bower-material/v1.1.5/angular-material.js

the links for js and css are broken, showing certificate errors in the browsers

Is anybody checking?

**Comment by themartorana** [comment]
Same for other JS files we use. It appears to be a site-wide certificate error.  The error is `SSL_ERROR_BAD_CERT_DOMAIN`  - the certificate being served does not list gitcdn.link among it's protected domains:

```
Websites prove their identity via certificates. Firefox does not trust this site because it uses a certificate that is not valid for gitcdn.link. The certificate is only valid for the following names: *.herokuapp.com, herokuapp.com
 
Error code: SSL_ERROR_BAD_CERT_DOMAIN
```

**Comment by schme16** [comment]
Hey all here's the longer version: https://github.com/schme16/gitcdn.xyz/issues/75#issuecomment-808803513

TL;DR: Cloudflare policy changed, gitcdn is getting flagged falsely, still working with cloudflare to sort the issue out. 
Apologies in the mean time for the outage and inconvenience!

**Comment by murtuzamj** [comment]
Okay. Its affecting apps published, cant change so easily in production environments.
I understood thats something to do with certificates.
